.github/renovate.json

```diff
@@ -1,16 +1,29 @@
 {
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "extends": [
-    "github>rancher/renovate-config#release"
+    "github>rancher/renovate-config//rancher-main#release"
   ],
   "baseBranches": [
-    "main"
+    "main",
+    "release/v0.12",
+    "release/v0.11",
+    "release/v0.10"
   ],
   "prHourlyLimit": 2,
-  "ignorePaths": [
-    "**/assets/**"
-  ],
-  "force": {
-    "constraints": {
+  "prConcurrentLimit": 4,
+  "prCreation": "status-success",
+  "packageRules": [
+    {
+      "matchBaseBranches": ["release/v0.12"],
+      "extends": ["github>rancher/renovate-config//rancher-2.11#release"]
```

Comment:
Are we using `#release` instead of `#rancher-presets` because https://github.com/rancher/renovate-config/pull/415 has been merged? :thinking: